src/GitElephant/Objects/Commit.php

```diff
@@ -269,25 +275,25 @@ private function parseOutputLines($outputLines)
      *
      * @return bool
      */
-    public function isRoot()
+    public function isRoot(): bool
     {
-        return count($this->parents) == 0;
+        return emtpy($this->parents);
```

Comment:
Sorry, typo crashing tests
```suggestion
        return empty($this->parents);
```